package.json

```diff
@@ -36,7 +36,8 @@
     "update-readme": "node scripts/buildtools update-readme",
     "retirejs": "retire --jspath target/js --package",
     "lint:ci": "yarn codegen && yarn lint",
-    "lint:esm": "yarn eslint --no-ignore --no-eslintrc --no-inline-config --config scripts/buildtools/eslint/.eslintrc.esm.js `find target/esm -type f -name \"*.js\"`",
+    "lint:esm": "eslint --no-ignore --no-eslintrc --no-inline-config --config scripts/buildtools/eslint/.eslintrc.esm.js `find target/esm -type f -name \"*.js\"`",
+    "lint:cdn": "eslint --no-ignore --no-eslintrc --no-inline-config --config scripts/buildtools/eslint/.eslintrc.cdn.js dist/dist/js/okta-sign-in.min.js",
```

Comment:
can we run this lint against all JS files under `/dist/js` ?

Reply:
we can select some of them, but maybe we don't want to against the `no-polyfill` one

Reply:
other file all give different lint errors, how about we take those ones as followups and focus on the cdn one first, since it's consumed by monolith directly